### DISCUSSION TITLE
Help and Note Symbols are away from side menu - Where are they?

### DISCUSSION THREAD

**Discussion by flowerhorse** [brunoherbelin/vimix#87] [General]
The Help and Note symbols are away from side menu - Where are they and why they're away?

**Comment by brunoherbelin** [comment]
To be more open to extension and associate with the list of windows, these actions are now in a sub-menu in the list of Windows of the left panel.

(see #75 for a screenshot)

**Comment by flowerhorse** [comment]
Optical it looks much better with only one sign there. But Help is important and not easy to find there. 
Maybe it could find place also in 'About Vimix?

**Comment by brunoherbelin** [comment]
Thanks for the suggestion: I added 'help' and 'logs' in the About dialog.

Also, sub-menu listing all available windows (the "..." icon) is now 'on mouse over', thus easy to discover.

**Comment by brunoherbelin** [comment]
With the idea to facilitate the access to help / notes, and in general of the menus, the version 0.8.1 has a new left panel :

![Screenshot from 2023-09-19 19-28-36](https://github.com/brunoherbelin/vimix/assets/60393189/2dbb4e2e-95ec-4247-97dc-62dcb3798831)

File / Edit and View menus are very standard to any app, so users should be familiar with it.
Then the control of the current session is clarified with a preview and unified representation (same as for sources).\

hope it helps!
